pallets/circuit/src/mock.rs

```diff
@@ -641,7 +641,7 @@ impl ExtBuilder {
                 b"insurance".to_vec(),
             ],
             confirm_events: vec![
-                b"ExecuteToken(executor,to,liquidity_token,amount_liquidity_token)".to_vec(),
+                b"ExecuteToken(_executor,to,liquidity_token,amount_liquidity_token)".to_vec(),
```

Comment:
what does it mean when a side effect has `_` prefixed? does it mean it's not used?

Reply:
Yeah it means that the argument is ignored during the confirmation step